### DISCUSSION TITLE
"JWT Expired" Error in Supabase Dashboard

### DISCUSSION THREAD

**Discussion by TheOtherBrian1** [supabase/supabase#21135] [Troubleshooting]
_This is a copy of a troubleshooting article on Supabase's docs site. It may be missing some details from the original. View the [original article](https://supabase.com/docs/guides/troubleshooting/jwt-expired-error-in-supabase-dashboard-F06k3x)._

When the Dashboard issues JWT expired errors, it usually means your computer's time is not in sync with the actual time.

Go to <https://time.is/> to compare your computer's clock with your local time. If your computer's time is inaccurate, sync it to resolve the issue

**Comment by afifurrohman-id** [comment]
still persistence.

**Comment by ghost** [comment]
I don’t understand a single thing in this app
Please can anyone explain

**Reply by uddinArsalan** [reply]
Refer to the docs 
[https://supabase.com/docs](url)

**Comment by aliutiev** [comment]
still here guys
